Polygon_mesh_processing/test/Polygon_mesh_processing/self_intersection_surface_mesh_test.cpp

```diff
@@ -81,6 +81,53 @@ int test_self_intersections(const char* filename,
   return 0;
 }
 
+template <typename K>
+int test_limited_self_intersections(const char* filename)
+{
+  typedef CGAL::Surface_mesh<typename K::Point_3>                Mesh;
+  typedef typename boost::graph_traits<Mesh>::face_descriptor    face_descriptor;
+
+  std::ifstream input(filename);
+  Mesh m;
+
+  if ( !input || !(input >> m) ) {
```

Comment:
All the IO efforts just to use old code... :>

Reply:
I blame the copy paste !